com.parsecgaming.parsec.yml

```diff
@@ -0,0 +1,74 @@
+---
+app-id: "com.parsecgaming.parsec"
+runtime: "org.freedesktop.Platform"
+runtime-version: "19.08"
+sdk: "org.freedesktop.Sdk"
+command: "parsec"
+copy-icon: true
+separate-locales: false
+finish-args:
+  - "--share=ipc"
+  - "--socket=wayland"
+  - "--socket=fallback-x11"
+  - "--socket=pulseaudio"
+  - "--share=network"
+  - "--device=dri"
+  - "--persist=."
+  - "--talk-name=org.freedesktop.Notifications"
+  - "--talk-name=org.freedesktop.portal.Fcitx"
+cleanup:
+  - "/include"
+  - "/lib/pkgconfig"
+  - "/lib/*.la"
+  - "/lib/*.a"
+  - "/share/pkgconfig"
+  - "/share/aclocal"
+  - "/share/gtk-doc"
+  - "/share/doc"
+  - "/share/info"
+  - "/share/man"
+  - "/man"
+modules:
+  - name: "parsec"
+    buildsystem: "simple"
+    build-commands:
+      - "install -Dm755 /usr/bin/ar -t ${FLATPAK_DEST}/bin"
+      - "install -Dm755 /usr/lib/$(gcc -print-multiarch)/libbfd-*.so -t ${FLATPAK_DEST}/lib"
+      - "install -Dm755 parsec.sh ${FLATPAK_DEST}/bin/parsec"
+      - "install -Dm755 apply_extra ${FLATPAK_DEST}/bin/apply_extra"
+      - "install -Dm644 com.parsecgaming.parsec.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png"
+      - "install -Dm644 com.parsecgaming.parsec.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop"
+      - "install -Dm644 com.parsecgaming.parsec.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml"
+
+    sources:
+      - type: "extra-data"
+        filename: "parsec-linux.deb"
+        only-arches:
+          - x86_64
+        url: "https://builds.parsecgaming.com/package/parsec-linux.deb"
+        sha256: "0c7207f419377fc35e30448acd00d8701300ac5a44936ac6d87a468c9afccdc1"
+        size: 3188156
+        x-checker-data:
+            type: "rotating-url"
+            url: "https://builds.parsecgaming.com/package/parsec-linux.deb"
+      - type: "script"
+        dest-filename: "apply_extra"
+        commands:
+          - "ar x parsec-linux.deb"
+          - "tar xf data.tar.xz"
+          - "mv usr/* ."
+          - "rm -rf usr debian-binary control.tar.gz data.tar.xz parsec-linux.deb"
+      - type: "script"
+        dest-filename: "parsec.sh"
+        commands:
+          - "[ ! -e $HOME/appdata.json ] && ln -s /app/extra/share/parsec/skel/appdata.json $HOME/appdata.json"
```

Comment:
What's the point of this given you install metainfo file?

Reply:
This is nothing about app metainfo. This is a file provided by parsec and required to be present in the home directory in order to start.